### ISSUE TITLE
Providing a prepend will only return the prepend

### ISSUE THREAD

**Issue by deevus** [danielcobo/uuid#1]
See here

https://github.com/danielcobo/uuid/blob/c02e40d1c691b1b42c497896bfde3d47ddfe7b2f/src/index.js#L76

Calling `uuid("id")` will only return `"id"`

**Comment by deevus** [comment]
It looks to be an issue with precedence (at least on my system)

**Comment by deevus** [comment]
```
Welcome to Node.js v14.19.0.
Type ".help" for more information.
> const uuid = require('@danielcobo/uuid')
undefined
> uuid('id')
'id'
> uuid()
'1g3svodcr00000ji3fsk19t3lkq5218a'
>
```

**Comment by deevus** [comment]
To make sure I wasn't crazy, here is a CodeSandbox

https://codesandbox.io/s/gracious-diffie-n1zbpb?file=/src/index.js

**Comment by danielcobo** [comment]
Thank you for submitting the issue. 
I've patched it in 1.0.2, see: https://codesandbox.io/s/happy-tdd-m0tcr0
